cleancat/base.py

```diff
@@ -258,7 +258,7 @@ def __init__(self, require_tld=True, default_scheme=None,
             )
 
     def clean(self, value):
-        if value == self.blank_value:
+        if value == self.blank_value and not self.required:
```

Comment:
Why do we need separate handling of this here? Why can't `super()` handle it?

Reply:
Indeed it was already handled by `super`. I removed this code and added an extra test to make sure I didn't break anything in ff3daca.